src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -263,14 +263,51 @@
         </div>
       </div>
       <div *ngIf="workflow?.descriptorType === WorkflowModel.DescriptorTypeEnum.Jupyter">
-        <a
-          target="_blank"
-          rel="noopener"
-          href="https://colab.research.google.com/github/{{ workflow.organization }}/{{ workflow.repository }}/blob/{{
-            selectedVersion.name
-          }}/{{ selectedVersion.workflow_path }}"
-          >Google Colab</a
-        >
+        <div fxLayout="column" fxLayoutAlign="space-between center" fxLayoutGap="0.625rem">
+          <a
+            mat-button
+            class="launch-with-button private-btn"
+            [matTooltip]="colabTooltip$ | async"
+            matTooltipPosition="left"
+            target="_blank"
+            rel="noopener"
+            [attr.href]="
+              config.COLAB_IMPORT_URL +
```

Comment:
If Colab and MyBinder only accept references to GitHub repos (?), then if we ever add notebooks support to BitBucket and/or Dockstore hosted, these links probably shouldn't exist or be disabled.

An extreme forwards looking edge case, as we only support GitHub today.